rotkehlchen/api/v1/encoding.py

```diff
@@ -1365,7 +1380,14 @@ def _transform_eth_address(
     except ValueError:
         # Validation will only let .eth names come here.
         # So let's see if it resolves to anything
-        resolved_address = ethereum.ens_lookup(given_address)
+        try:
+            resolved_address = ethereum.ens_lookup(given_address)
+        except (RemoteError, InputError) as e:
+            raise ValidationError(
+                f'Given ENS address {given_address} could not be resolved due to: {str(e)}',
+                field_name='address',
```

Comment:
the other errors for ethereum also do not mention the chain. That is because at first we had ethereum only ENS. We now have also bitcoin and kusama. We should probably add the "chain" attribute in all the errors of ethereum too.

Reply:
But changed that as discussed.